code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -425,6 +425,14 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 			pref.rlimb_data.Cut()
 			pref.body_markings.Cut()
 
+			pref.culture = "[pick(mob_species.possible_cultures)]"
+			var/decl/origin_item/culture/OC = decls_repository.get_decl(text2path(pref.culture))
+			pref.origin = "[pick(OC.possible_origins)]"
```

Comment:
These `picks` should instead pick a low-effort default culture, to avoid new players appearing as a difficult-to-RP culture because they didn't set up all their prefs. 

Origin and culture should always be a deliberate choice.

Reply:
See above.